resources/i18n/pl_PL/uranium.po

```diff
@@ -37,13 +37,13 @@ msgstr "Plik Wavefront OBJ"
 #: /home/ruben/Projects/Uranium/plugins/FileHandlers/OBJWriter/OBJWriter.py:23
 msgctxt "@error:not supported"
 msgid "OBJWriter does not support non-text mode."
-msgstr ""
+msgstr "Zapisywacz OBJ Nie obsługuje trybu nietekstowego."
```

Comment:
Should "Nie" (which means "not") not be written lowercase?

Reply:
Yes, that's correct, it should be "Zapisywacz OBJ nie obsługuje trybu nietekstowego.", just a typo

Reply:
I decide to keep the name "OBJWriter", because this is a plugin name and it shouldn't change.

Reply:
Of course, in my opinion it looks better, I translated it that way to keep the manner of translating "xxxWriter" and "xxxReader" plugins names as they were translated previously in Cura repository. I can revise and change all these names in Cura repository if you wish

Reply:
@diegopradogesto Well, then "Zapisywacz OBJ" -> "OBJWriter".
@drzejkopf Right?

But I see the same also in other languages. Like Russian, if I'm not wrong. Just remember that I saw something like this.